thirdeye/thirdeye-frontend/app/serializers/performance.js

```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  /*
+   * @summary normalizing the payload from api response simpler or legacy json that may not support the http://jsonapi.org/ spec
+   */
+  normalizeQueryResponse(store, primaryModelClass, payload) {
```

Comment:
Can we also document the params in the jsdoc?

Reply:
Yes! We should. Will do that. Thanks!

Reply:
Actually this is a built in hook. Like `model`, `preModel` hooks in route. But i can explicitly document it as well.

Reply:
/**
    * @summary A serializer hook. Normalizing the payload from api response simpler or legacy json that may not support the http://jsonapi.org/ spec
    * @method normalizeQueryResponse
    * @param {DS.Store} store
    * @param {DS.Model} primaryModelClass
    * @param {Object} payload - the data object
    * @return {Obect}
    */